tools/sof-kernel-log-check.sh

```diff
@@ -345,6 +345,15 @@ ignore_str="$ignore_str"'|iwlwifi [[:digit:].:]+: '
 # SDW related logs
 #
 
+#Irrelated log brought in verify-kernel-boot-log.sh
+#http://sof-ci.sh.intel.com/#/result/planresultdetail/4120?model=CML_RVP_HDA&testcase=verify-kernel-boot-log
+ignore_str="$ignore_str"'|Failed to mount NFSD configuration filesystem'
```

Comment:
There is something wrong with this particular system, we shouldn't trying to setup anything NFS related on test systems. Please try `sudo apt remove nfs-common` first and see if this resolves this issue.

Reply:
yes , sudo apt remove nfs-common first helps resolve this issue. :)